src/commons/utils/jira_utils.py

```diff
@@ -189,13 +188,6 @@ def get_all_tests_details_from_tp(self, tp_id: str, reset_status: bool = False)
                                     f" TE '{texe['key']}'")
                 tests_dict[test['key']] = test
                 tests_dict[test['key']]["te"] = texe
-                if reset_status:
```

Comment:
Why this code is removed? This is optional and parameterized so that user can decide to reset jira status or not.